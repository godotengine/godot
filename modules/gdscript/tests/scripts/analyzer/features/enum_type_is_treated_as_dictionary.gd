enum MyEnum {
	ZERO,
	ONE,
	TWO,
}

func test():
	for key in MyEnum.keys():
		prints(key, MyEnum[key])

	# https://github.com/godotengine/godot/issues/55491
	for key in MyEnum:
		prints(key, MyEnum[key])
