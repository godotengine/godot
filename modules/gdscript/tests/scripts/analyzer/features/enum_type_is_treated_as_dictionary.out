GDTEST_OK
ZERO 0
ONE 1
TWO 2
ZERO 0
ONE 1
TWO 2
